dpc/src/account/account_format.rs

```diff
@@ -15,6 +15,8 @@
 // along with the snarkVM library. If not, see <https://www.gnu.org/licenses/>.
 
 pub static ACCOUNT_ENCRYPTION_AND_SIGNATURE_INPUT: &str = "AleoAccountEncryptionAndSignatureScheme0";
+pub static ACCOUNT_SEED_SK_SIG_DOMAIN: &str = "AleoAccountSeedSignatureSecretKey0";
+pub static ACCOUNT_SEED_R_SIG_DOMAIN: &str = "AleoAccountSeedSignatureRandomizer0";
```

Comment:
Maybe this should be something like 
`pub const ACCOUNT_blah_blah: &'static [u8] = b"AleoAccountBlahBlahBlah"`?